lib/mix/lib/mix.ex

```diff
@@ -172,25 +172,31 @@ defmodule Mix do
       echo "world!"
 
       # mix.exs
-      defp create_aliases do
+      defp aliases do
         [
-          "taskalias": ["hex.info", "run priv/hello.exs", "cmd priv/world.sh"],
-          "taskalias2": ["run priv/hello1.exs", "run priv/hello2.exs"]
+          task_alias: ["hex.info", "run priv/hello.exs", "cmd priv/world.sh"],
```

Comment:
Nitpick: maybe call these `task_alias1` and `task_alias2`? Kind of looks weird with one with the index and the other one without it.

Reply:
We usually don't add underscores to tasks so maybe just `task` or `alias` would be better?

Reply:
both suggestions have been implemented